c-scape/src/ctype/lsb_abi/ctype_b_loc.rs

```diff
@@ -0,0 +1,399 @@
+use super::SyncWrapper;
+
+pub const fn bsw(v: u16) -> u16 {
+    u16::from_be(v)
+}
```

Comment:
If rustfmt is going to put each element on its own line, would it make sense to just inline this?